lib/apiservers/engine/backends/container.go

```diff
@@ -69,6 +70,13 @@ const (
 	MinCPUs = 1
 	// DefaultCPUs - the default number of container VM CPUs
 	DefaultCPUs = 2
+
+	// Random numbers generated by fair dice roll :)
+	DefaultInitialInterval     = 2 * time.Second
```

Comment:
They are randomly picked by me, you can experiment with them via

```
package main

import (
        "fmt"
        "time"

        "github.com/cenkalti/backoff"
)

const (
        DefaultInitialInterval     = 2 * time.Second
        DefaultRandomizationFactor = 0.5
        DefaultMultiplier          = 2
        DefaultMaxInterval         = 30 * time.Second
        DefaultMaxElapsedTime      = 1 * time.Minute
)

type Clock struct {
        now time.Time
}

func (c *Clock) Now() time.Time {
        return c.now
}

func (c *Clock) Sleep(i time.Duration) {
        c.now = c.now.Add(i)
}
func main() {
        clock := &Clock{now: time.Now()}
        b := &backoff.ExponentialBackOff{
                InitialInterval:     DefaultInitialInterval,
                RandomizationFactor: DefaultRandomizationFactor,
                Multiplier:          DefaultMultiplier,
                MaxInterval:         DefaultMaxInterval,
                MaxElapsedTime:      DefaultMaxElapsedTime,
                Clock:               clock,
        }
        // Reset the interval back to the initial retry interval and restart the timer
        b.Reset()
        for {
                next := b.NextBackOff()
                if next == backoff.Stop {
                        break
                }
                fmt.Printf("[%s]\n", next)
                clock.Sleep(next)
        }
}
```

running them 3 times shows following distribution

```
[conur@vir:/opt/go/src/github.com/vmware/vic(retry)] go run retry.go
[2.209320576s]
[5.762036353s]
[9.316480426s]
[15.003426995s]
[27.739124912s]
[conur@vir:/opt/go/src/github.com/vmware/vic(retry)] go run retry.go
[2.209320576s]
[5.762036353s]
[9.316480426s]
[15.003426995s]
[27.739124912s]
[conur@vir:/opt/go/src/github.com/vmware/vic(retry)] go run retry.go
[2.209320576s]
[5.762036353s]
[9.316480426s]
[15.003426995s]
[27.739124912s]
```